perf.h

```diff
@@ -35,6 +35,10 @@ class Perf {
   void ReportResults();
 
  private:
+  // Compute the FPS every 100 frames and write it to the screen.
```

Comment:
To be more clear, this method just calculates the FPS and writes it to the screen.

The "100 frames" is a variable(print_freq_in_frames_), so don't mention that. Also, this method doesn't check if 100 frames have passed or not, it just prints to the screen directly.

Just pointing out that if I looked at this comment as the documentation of this function, it would be misleading.